zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -56,6 +59,17 @@
 }
 
 
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
+
+    attributes = {
+        0x0009: ("mode", types.uint8_t, True),
+    }
```

Comment:
Ideally, we would use the new-style zigpy `AttributeDefs` here instead.
Can you try and see if something like this still presents the attribute in the clusters menu?

```python 
    class AttributeDefs(BaseAttributeDefs):
        """Attribute definitions."""

        mode = ZCLAttributeDef(
            id=0x0009,
            type=types.uint8_t,
            is_manufacturer_specific=True,
        )
```

Also, for what is that attribute used? So, what behavior does the switch change when it's `0x01` instead of `0x00`?

Reply:
removed still works